дз1ахлюстина/дз1ахлюстина/Program.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace дз1ахлюстина
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            double x;
+            Console.WriteLine("Введите длину ребра октаэдра:");
+            x = double.Parse(Console.ReadLine());
+
+            double S;
+            S = 2 * Math.Pow(x,2) * Math.Sqrt(3);
+            S = Math.Round(S, 3);
+            Console.WriteLine("Площадь поверхности октаэдра:" + S);
+
+            double V;
+            V = 0.333 * Math.Sqrt(2) * Math.Pow(x,3);
```

Comment:
А это уже **ошибка**. Число 0.333 грубо приближает дробь 1/3. В задании не указана точность вычислений, поэтому их нужно производить с максимально возможной точностью.
```suggestion
            V = Math.Sqrt(2) * Math.Pow(x,3) / 3;
```
В формулах нельзя использовать округленные константы.